specs/sharding/p2p-interface.md

```diff
@@ -77,9 +80,9 @@ def compute_subnet_for_shard_blob(state: BeaconState, slot: Slot, shard: Shard)
 The following validations MUST pass before forwarding the `signed_blob`,
 on the horizontal subnet or creating samples for it. Alias `blob = signed_blob.message`.
 
-- _[IGNORE]_ The `blob` is not from a future slot (with a `MAXIMUM_GOSSIP_CLOCK_DISPARITY` allowance) --
-  i.e. validate that `blob.slot <= current_slot`
-  (a client MAY queue future blobs for processing at the appropriate slot).
+- _[IGNORE]_ The `blob` is published 1 slot early or later (with a `MAXIMUM_GOSSIP_CLOCK_DISPARITY` allowance) --
```

Comment:
One slot because the proposer moves at N - 1/3, meaning the builder should move *immediately* which might be marginally before N?

Reply:
exactly